src/Extend/Sortmap.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extend;
+
+use Flarum\Extension\Extension;
+use Illuminate\Contracts\Container\Container;
+
+class Sortmap implements ExtenderInterface
```

Comment:
Let's drop this class for now until we figure out a cleaner way to generalize this for all models. Other than that, good to go IMO.